apps/funke/app.config.js

```diff
@@ -6,14 +6,17 @@ const variants = {
   development: {
     bundle: '.dev',
     name: ' (Dev)',
+    mediatorDid: 'did:web:mediator.dev.paradym.id',
```

Comment:
Funke has no didcomm setup, so this is not needed I think. did you get an error?